aviary/aviary.py

```diff
@@ -598,23 +598,66 @@ def main():
         default=3
     )
 
+    binning_group.add_argument(
+        '--extra-binners', '--extra_binners', '--extra-binner', '--extra_binner',
+        help='Optional list of extra binning algorithms to run. Can be any combination of: \n'
+             'maxbin, maxbin2, concoct \n'
+             'These binners are skipped by default as they can have long runtimes'
+             'Capitals will be auto-corrected. N.B. specifying "maxbin" and "maxbin2" are equivalent \n',
```

Comment:
Is this true even when capitalized versions aren't in choices?

Reply:
Good catch, choices makes this obsolete now